node_build/make.js

```diff
@@ -116,6 +116,10 @@ Builder.configure({
         }
     }
 
+    if (process.env['NO_TEST'] === '1') {
```

Comment:
Why given https://github.com/tigusoft/cjdns/blob/pr-build-options-skip-tests-clean/node_build/make.js#L437 causes the tests not to be executed anyway ?